addons/l10n_fr_fec/tests/test_wizard.py

```diff
@@ -66,15 +66,51 @@ def setUpClass(cls, chart_template_ref='l10n_fr.l10n_fr_pcg_chart_template'):
         })
         cls.invoice_a.action_post()
 
+    def _get_expected_report(self):
```

Comment:
If partner_a.id won't change, I think that you can use a class variable instead of a function :p

Reply:
Indeed. Changed